src/app/cases/page.tsx

```diff
@@ -44,73 +54,90 @@ export default function Page() {
     });
   }, []);
 
+  const resetFilters = () => {
+    setCaseFilters({
+      remote: new Set(),
+      role: new Set(),
+      agency: new Set(),
+      languages: new Set(),
+      countries: new Set(),
+    });
+  };
+
   return (
     <PageContainer>
-      <div>
-        <H2>Browse Available Cases</H2>
-        <FiltersContainer>
-          <FilterDropdown
-            defaultValue="Remote/In Person"
-            options={['Remote/In Person', 'Remote Only', 'In Person Only']}
-            onChange={v =>
-              setCaseFilters({ ...caseFilters, remote: v as string })
-            }
-          />
-          <FilterDropdown
-            defaultValue="Attorney/Interpreter"
-            options={['Attorney/Interpreter', 'Interpreter Only']}
-            onChange={v =>
-              setCaseFilters({ ...caseFilters, role: v as string })
-            }
-          />
-          <FilterDropdown
-            defaultValue="Anywhere"
-            // requires knowing user location
-            options={[
-              'Within 5 miles',
-              'Within 10 miles',
-              'Within 20 miles',
-              'Within 100 miles',
-              'Anywhere',
-            ]}
-            onChange={v =>
-              setCaseFilters({ ...caseFilters, distance: v as string })
-            }
-          />
-          <FilterDropdown
-            defaultValue="All countries"
-            multi
-            options={[
-              'All countries',
-              ...Array.from(
-                new Set(
-                  caseData
-                    .filter(c => c.country)
-                    .map(c => (c.country ? c.country : '')),
-                ),
-              ),
-            ]}
-            onChange={v =>
-              setCaseFilters({ ...caseFilters, countries: v as string[] })
-            }
-          />
-        </FiltersContainer>
-      </div>
+      <H2>Browse Available Cases</H2>
+      <FiltersContainer>
```

Comment:
question on filters functionality (cc @davidqing6432)

If we select a card on the left, and its interest form is pulled up on the right, but then we filter out that card, it disappears from the left but its interest form is still pulled up on the right. Do we want to change this so that the interest form is reset to be the one corresponding to the first card on the right, post-filtering? I think it's kind of weird to have an interest form pulled up that doesn't correspond to a card.

The only issue is if someone is in the middle of filling out a form, and then filters the cards, their progress will be lost (assuming we don't implement the stretch feature of saving in-progress form responses) but tbh idk why they would do that

Reply:
not david, but i recall having this discussion and i think we settled on keeping the selected case info, not sure the specific reason why tho. i can see it going either way.

Reply:
should be we leave the selected case info up, and then you can reselect something if you want

Reply:
resolved: keep selected case info, but make card selected style more prominent -- ie change color